<launch>

  <arg name="model" default="$(find iiwa_description)/urdf/iiwa14-with-cam.xacro" />
  <arg name="gui" default="false" />
  <!--arg name="rvizconfig" default="$(find hddtable)/rviz/urdf.rviz" /-->

  <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
  <param name="use_gui" value="$(arg gui)"/>

  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
     <rosparam param="source_list">["/kit_gripper/joint_state"] </rosparam>
  </node>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
  <node name="rviz" pkg="rviz" type="rviz" required="true" />

</launch>
